Tests/SentryTests/Integrations/Session/SentrySessionTrackerTests.swift

```diff
@@ -15,7 +15,7 @@ class SentrySessionTrackerTests: XCTestCase {
 
         let notificationCenter = TestNSNotificationCenterWrapper()
         let dispatchQueue = TestSentryDispatchQueueWrapper()
-        lazy var fileManager = try! SentryFileManager(options: options, dispatchQueueWrapper: dispatchQueue)
+        lazy var fileManager = try! SentryFileManager(options: options, dispatchQueueWrapper: SentryDispatchQueueWrapper())
```

Comment:
Why are we switching this to a live dispatch queue wrapper?

Reply:
Good catch. That's not intended. I forgot to undo that change.